README.md

```diff
@@ -1,8 +1,234 @@
+
+### Build instructions
+To build the binary for the IPC agent you need to build the requirements in your environment, clone this repo, and build the binary following these steps:
+```bash
+$ git clone https://github.com/filecoin-project/ref-fvm.git
+$ cd ipc-agent
+$ rustup target add wasm32-unknown-unknown
+$ make build
+```
+
+This builds the binary of the IPC agent in the `./bin` folder of your repo. If you want to make the command available everywhere, add this folder to the binary `PATH` of your system. To see if the installation was successfully you can run the following command: 
+``` bash
+$ ./bin/ipc_agent --help
+
+The IPC agent command line tool
+
+Usage: ipc_agent [OPTIONS] <COMMAND>
+
+Commands:
+  daemon                Launch the ipc agent daemon
+  reload-config         Config commands
+  init-config           Arguments to initialize a new empty config file
+  create-subnet         Subnet manager commands
+  list-subnets          List child subnets
+  join-subnet           Join a subnet
+  leave-subnet          Leaving a subnet
+  kill-subnet           Kill an existing subnet
+  fund                  Send funds from a parent to a child subnet
+  release               Release operation in the gateway actor
+  propagate             Propagate operation in the gateway actor
+  whitelist-propagator  Whitelist propagators in the gateway actor
+  send-value            Send value to an address within a subnet
+  wallet-new            Create new wallet in subnet
+  help                  Print this message or the help of the given subcommand(s)
+
+Options:
+  -c, --config-path <CONFIG_PATH>  The toml config file path for IPC Agent, default to ${HOME}/.ipc_agent/config.toml
+  -h, --help                       Print help
+  -V, --version                    Print version
+```
+
+## Infrastructure
+IPC currently uses [a fork of Lotus](https://github.com/consensus-shipyard/lotus), that we like to call _Eudico_, to run its subnets. The IPC agent does nothing by itself, and is just an orchestrator over existing subnet deployments. To ease the deployment of new subnets and nodes, we provide a set of convenient scripts to deploy all the infrastructure required to run IPC. 
+
+### Install infrastructure scripts
+All the infrastructure scripts assume live in [Eudico's repo](https://github.com/consensus-shipyard/lotus/tree/spacenet/scripts/ipc), and assume a working installation of Docker. To install Docker [follow this link])(https://docs.docker.com/get-docker/) and choose your working environment.
+
+With Docker installed, you can `make install-infra` to have all the infrastructure scripts available locally. This make rule will build the docker image that you need to run subnets, and install the infrastructure scripts in the `./bin` folder.
```

Comment:
```suggestion
[Eudico](https://github.com/consensus-shipyard/lotus/tree/spacenet/scripts/ipc) provides a set of infrastructure scripts, which assume a working installation of Docker. To install Docker [follow this link])(https://docs.docker.com/get-docker/) and choose your working environment.

With Docker installed, you can then `make install-infra` in this repository to clone the eudico repo, build the docker image that you need to run subnets, and install the infrastructure scripts in the `./bin` folder.
```

This read a bit confusing. You're saying all scripts live in the Eudico repo, so the natural thing to do after reading this would be to go on the eudico/lotus repo and `make install-infra` there even though no instructions to that effect are provided, but then it turns out the make rule for ipc-agent handles all of this.